README.md

```diff
@@ -2,38 +2,75 @@
 
 The best way to build an API, now for Heroku.
 
-Make a git repo with anything in it. The repo is just a pretext to
-push to Heroku.
+Updated for PostgREST v7.0.1.
 
-Note that **the free tier Heroku PostgreSQL addon will not work** because it
-does not support having multiple database roles. **Heroku Postgres** paid tiers do support multiple
-database roles though you'll have to create them through [Heroku Postgres Credentials](https://devcenter.heroku.com/articles/heroku-postgresql-credentials).
+NOTE: **The free tier Heroku PostgreSQL addon will not work** because it does
+not support having multiple database roles. **Heroku Postgres** paid tiers do
+support multiple database roles, though you'll have to create them through
+[Heroku Postgres
+Credentials](https://devcenter.heroku.com/articles/heroku-postgresql-credentials).
 
-We recommend spinning up a Postgres database with [Amazon
-RDS](https://aws.amazon.com/rds/). Make sure you create it in
-the **Virginia region** because that's where Heroku's dynos are
-and it will decrease latency between the server and db.
+We recommend spinning up a Postgres database with
+[ElephantSQL](https://www.elephantsql.com/) for free tier usage.
```

Comment:
How about making ElephantSQL an alternative to RDS? You could put it below on another subheading.

I think RDS should stay because most users are familiar with it for production.